templates/ruby/lib/container/file.rb.twig

```diff
@@ -0,0 +1,9 @@
+module {{spec.title | caseUcfirst}}
+    class AppwriteFile
```

Comment:
Avoid the word `Appwrite` inside the templates. You can use a Twig param to produce in the generated code.

Reply:
Because we already use {{ spec.title | caseDash }} as a namespace we might be able to use just `File` for the class name.

Reply:
@eldadfux File will clash with the File keyword that already exists in Ruby lang for file operations

Reply:
@eldadfux If i named the class File then while accessing can I use Module(Appwrite)::File.new() instead of File.new() ?

Reply:
I'm not sure, please test it. Worst case we can use the Appwrite keyword but auto-generate it using the twig param and not have it hardcoded in the template.

Reply:
Okay!